path_2024.py

```diff
@@ -81,7 +82,8 @@ def run() -> int:
         print("Error: upload_commands")
         return -1
 
-    while True:
+    start_time = time.time()
+    while time.time() - start_time < STOP_CONDITION_TIMEOUT:
```

Comment:
Is this tested working? Maybe add a comment stating that this is tested with a modified timeout value (e.g. 5 seconds).

Reply:
would this have to be tested in mission planner or can I test this locally on my computer?

Reply:
You can download Mission Planner on your local computer. Then you can complete the following steps to run a simulation on Mission Planner: https://uwarg-docs.atlassian.net/wiki/spaces/CV/pages/2240643073/Mission+Planner+Simulated+Ardupilot#MAVLink-forwarding. You can then test the pathing code on the simulator and see if the code you added works.

Reply:
I have a macbook, so I can't install mission planner on my computer